forem/forem
# Fix: Modal appears under the navbar #13195

## What type of PR is this? (check all applicable)


## Description
fixes #13195 , where I made the modal appear in the downward direction by adding 2 classes to dropdown element

## Related Tickets & Documents

## QA Instructions, Screenshots, Recordings

Create a draft post and then click on 3 dots on the left(show more dropdown), now you can see dropdown appear in downward direction.
Tested on brave browser and mozilla firefox on ubuntu 20.04
![Screenshot from 2021-07-15 04-15-39](https://user-images.githubusercontent.com/59141533/125703102-6f827e3b-ac29-4d86-882c-69722161cf15.png)


### UI accessibility concerns?

No it wont effect user accessibility

## Added/updated tests?

- [x] No, and this is why: As this is just a change in direction of dropdown, there wont be any test cases to test.

## [Forem core team only] How will this change be communicated?


      [Admin Guide](https://admin.forem.com/), or
      [Storybook](https://storybook.forem.com/) (for Crayons components)
      or in a [forem.dev](http://forem.dev) post
- [x] This change does not need to be communicated, and this is why not: As this is just change in direction of dropdown, it wont effect others parts of code or UI.

Screenshots on different resolutions:
![Screenshot from 2021-07-15 04-17-24](https://user-images.githubusercontent.com/59141533/125703448-f1bf7165-3d95-490e-8e13-b6a5d663e0cd.png)
![Screenshot from 2021-07-15 04-17-37](https://user-images.githubusercontent.com/59141533/125703460-56114e06-a407-49ae-a2b2-cf9426271487.png)
![Screenshot from 2021-07-15 04-17-57](https://user-images.githubusercontent.com/59141533/125703473-34935c34-5813-4826-8e0c-574514e9a113.png)
